cargo-dist/src/config.rs

```diff
@@ -398,6 +407,9 @@ impl DistMetadata {
         if publish_jobs.is_none() {
             *publish_jobs = workspace_config.publish_jobs.clone();
         }
+        if publish_prereleases.is_none() {
+            *publish_prereleases = workspace_config.publish_prereleases;
+        }
```

Comment:
We've really gotta stop merging things into packages if we're not gonna respect it (we only respect the workspace setting).

Reply:
Should I just remove this?

Reply:
change it to the other pattern where we warn that we're ignoring it